### PR TITLE
Symbol description

### PR DESCRIPTION
**Pull request by TadeasKucera**
Symbols are here optionally assigned a description which is then used in error messages instead of name of the symbol. The description can be assigned in TokenBuilder. The behaviour when _description is assigned in the TokenBuilder but no symbol is created is the same as if the description was not assigned at all.

### CODE REVIEW

File: include/pog/token_builder.h [hunk 1]
```diff
@@ -33,6 +33,8 @@ class TokenBuilder
 				const auto& prec = _precedence.value();
 				symbol->set_precedence(prec.level, prec.assoc);
 			}
+			if(symbol && _description.size() != 0)
```
Comment (metthal) [inline include/pog/token_builder.h:37]: Empty line is missing above this one.

File: tests/test_token_builder.cpp [hunk 2]
```diff
@@ -54,6 +54,21 @@ SingleTokenWithSymbol) {
 	EXPECT_EQ(tokenizer.get_tokens()[1]->get_pattern(), "abc");
 }
 
+TEST_F(TestTokenBuilder,
+SingleTokenWithDescription) {
+	TokenBuilder<int> tb(&grammar, &tokenizer, "abc");
+	tb.symbol("ABC");
+	tb.description("abc token");
+	tb.done();
```
Comment (metthal) [inline tests/test_token_builder.cpp]: I would personally like if you created one more token here without description and then in test compared that description is equal to name.

**Review by metthal — CHANGES_REQUESTED** [review]

File: include/pog/token_builder.h (hunk 1 above)
Comment (TadeasKucera) [reply include/pog/token_builder.h:37]: The missing line has been added.

File: tests/test_token_builder.cpp (hunk 2 above)
Comment (TadeasKucera) [reply tests/test_token_builder.cpp]: Another token 'DEF' has been added and the test was renamed to MultipleTokensDescription. Is it ok?

**Review by metthal — APPROVED** [review]

**Pull request merged**